### ISSUE TITLE
bower.json is outdated

### ISSUE THREAD

**Issue by eko24** [rubentd/gifplayer#24]
Please update/publish your bower.json file, because I spent a hell of a lot time to notice this
```
"description": "Customizable jquery plugin to play and stop animated gifs. Similar to 9gag's. Supports video",
  "version": "0.3.2",
  "main" : [
    "dist/jquery.gifplayer.js",
    "dist/gifplayer.css"
  ]
```

Thank you

**Comment by rubentd** [comment]
Sorry, what do you mean? 0.3.2 is the latest version

**Comment by eko24** [comment]
Right, but when I installed it with `bower install --save jquery.gifplayer` yesterday the folder structure was like this:

```
dist/
  - jquery.gifplayer.js
  - css/gifplayer.css
```

But as you can see `main` variable in downloaded bower.json containts different path:
```
"main" : [
    "js/jquery.gifplayer.js",
    "css/gifplayer.css"
  ],
```

Perhaps it needs to be republished on bower package manager or so ?

**Comment by rubentd** [comment]
You're right, the registered bower.json file has not been updated, is different than the one in the repo, I'll try bumping the version number

**Comment by eko24** [comment]
Thank you !
I guess you might close issue when you'll update it
